blacklight-spotlight.gemspec

```diff
@@ -20,7 +20,7 @@ these collections.)
   s.required_ruby_version = '~> 2.2'
 
   s.add_dependency 'rails', '~> 4.0', '>= 4.2.0'
-  s.add_dependency 'blacklight', '~> 6.0'
+  s.add_dependency 'blacklight', '~> 6.3'
```

Comment:
There were a couple backwards-incompatible changes introduced in Blacklight 6.3. I've submitted PRs upstream, but in the meantime have adopted the new approach instead.

- https://github.com/projectblacklight/blacklight/pull/1430
- https://github.com/projectblacklight/blacklight/pull/1429